### PR TITLE
add client example for elementwise_normalization

### PR DESCRIPTION
**Pull request by guangzlu**
Added client example for elementwise_normalization, for Pytorch use.

### CODE REVIEW

File: client_example/10_elementwise_normalization/elementwise_layernorm2d.cpp [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+int main()
+{
+    bool time_kernel = true;
+
+    ck::index_t M      = 48 * 256;
+    ck::index_t N      = 1024;
+    ck::index_t Stride = N;
+
+    auto xy_size = (M - 1) * Stride + N;
```
Comment (rocking5566) [inline client_example/10_elementwise_normalization/elementwise_layernorm2d.cpp:53]: xy_size > mn_size

File: client_example/10_elementwise_normalization/elementwise_layernorm2d.cpp [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+    SimpleDeviceMem a_dev_buf(sizeof(ADataType) * xy_size);
+    SimpleDeviceMem b_dev_buf(sizeof(BDataType) * xy_size);
+    SimpleDeviceMem gamma_dev_buf(sizeof(GammaDataType) * N);
+    SimpleDeviceMem beta_dev_buf(sizeof(BetaDataType) * N);
+    SimpleDeviceMem y_dev_buf(sizeof(YDataType) * xy_size);
+
+    std::array<const void*, 2> input = {a_dev_buf.GetDeviceBuffer(), b_dev_buf.GetDeviceBuffer()};
+    std::vector<ck::index_t> inputStride                 = {Stride, 1};
+    std::array<std::vector<ck::index_t>, 2> inputStrides = {inputStride, inputStride};
```
[the remark this answers is not in the document]
Comment (guangzlu) [reply client_example/10_elementwise_normalization/elementwise_layernorm2d.cpp]: Thanks a lot. It does make sense. I have fixed them.

File: client_example/10_elementwise_normalization/elementwise_layernorm2d.cpp [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+    auto mn_size = (M - 1) * Stride + N;
+
+    SimpleDeviceMem a_dev_buf(sizeof(ADataType) * mn_size);
+    SimpleDeviceMem b_dev_buf(sizeof(BDataType) * mn_size);
+    SimpleDeviceMem gamma_dev_buf(sizeof(GammaDataType) * N);
+    SimpleDeviceMem beta_dev_buf(sizeof(BetaDataType) * N);
+    SimpleDeviceMem y_dev_buf(sizeof(YDataType) * mn_size);
+
+    std::array<const void*, 2> input  = {a_dev_buf.GetDeviceBuffer(), b_dev_buf.GetDeviceBuffer()};
```
Comment (rocking5566) [inline client_example/10_elementwise_normalization/elementwise_layernorm2d.cpp:61]: ab_input or ab

**Review by rocking5566 — APPROVED** [review]
LGTM

**Pull request merged**